### PR TITLE
MpiIsoApp 24.05.4: fix warning

### PR DESCRIPTION
**Pull request by mgross**
See https://github.com/Pandora-IsoMemo/iso-app/issues/199

----

# MpiIsoApp 24.05.4
## Bug Fixes
- fix issue with warning whenever using a bayesian model (#199)

### CODE REVIEW

File: R/01-estimateMap.R [hunk 1]
```diff
@@ -1966,6 +1967,14 @@ modelSpread <- function(data, K, iter, burnin, MinMax, smoothConst, penalty,
                   (XX %*% betamc[usedsamples[x], ]) * sRe + mRe), 1, var))))))
 }
 
+updatePenalty <- function(penalty, bs) {
+  if (bs == "ds" & penalty == 1) {
+    penalty <- c(1, 0.5)
+  }
```
Comment (arunge) [inline R/01-estimateMap.R:1973]: @mgross I extracted your code into a separate function `updatePenalty()`. Additionally, Github copilot suggests to add following cases. Do these cases make any sense? Is it worth to add them?


```suggestion
 } else if (bs == "ds" & penalty == 2) {
    penalty <- c(1, 0.5, 0.25)
  } else if (bs == "sos" & penalty == 1) {
    penalty <- c(1, 0.5)
  } else if (bs == "sos" & penalty == 2) {
    penalty <- c(1, 0.5, 0.25)
  }
```

**Review by arunge — CHANGES_REQUESTED** [review]
@mgross Thanks for your fix! As mentioned below, I have one question left. Can you check, please?

File: R/01-estimateMap.R (hunk 1 above)
Comment (mgross) [reply R/01-estimateMap.R:1973]: Nein, das ist kompletter Quatsch :)

Comment (arunge) [reply R/01-estimateMap.R:1973]: Ja, war fast zu erwarten, viele Vorschläge waren bisher komplett daneben, obwohl einige wenige durchaus gut sind ... :sweat_smile:

**Review by arunge — APPROVED** [review]

**Pull request merged**